tests/common.py

```diff
@@ -253,27 +208,14 @@ def assert_equal_vectors(self, truth, test, rtol):
     def compare_forces(
```

Comment:
Simplified `compare_forces` to accept an unbound GPU impl (rather than constructing it in the function), removing the need to loop over parameter sets for efficiency.